src/docfx/Models/WkhtmltopdfJsonConfig.cs

```diff
@@ -12,10 +13,18 @@ namespace Microsoft.DocAsCode
     [Serializable]
     public class WkhtmltopdfJsonConfig
     {
+        /// <summary>
+        /// Gets or sets the path and file name of a wkhtmltopdf.exe compatible executable.
+        /// </summary>
+        [JsonProperty("filePath")]
+        public string FilePath { get; set; }
+
         /// <summary>
         /// Specify additional command line arguments that should be passed to the wkhtmltopdf executable.
         /// </summary>
         [JsonProperty("additionalArguments")]
         public string AdditionalArguments { get; set; }
+
+        internal string FullFilePath => string.IsNullOrEmpty(FilePath) ? FilePath : Path.GetFullPath(FilePath);
```

Comment:
There are 2 cases here:
1. If `FilePath` is already full path, we can use it directly.
2. If `FilePath` is a relative path, we should use `Path.Combile(BaseDirectory, FilePath)`. `BaseDirectory` is where `docfx.json` exists. You can use `PdfJsonConfig.BaseDirectory` to get this value.

Consider some usage like `docfx subfolder/docfx.json`, where the current directory is not a base directory, the `Path.GetFullPath(FilePath)` will go to a wrong place.